suite-native/state/src/index.ts

```diff
@@ -1 +1,2 @@
 export * from './store';
```

Comment:
Do we need to export store? I think we do not because it's wrapped in SecureStoreProvider now.

Reply:
fixed in de5504964